setup.py

```diff
@@ -13,11 +15,10 @@
     changelog.read_text()
 )
 long_description_content_type = 'text/x-rst'
-version = here / 'VERSION'
 
 setup(
     name='juju',
-    version=version.read_text().strip(),
+    version=CLIENT_VERSION.strip(),
```

Comment:
Do we still need to strip?

Reply:
We currently don't need to, but the line that's coming from will be manually edited by people who're making releases (replaces the file VERSION) in the future, so I thought this would cleanup any silly mistakes like "4.2.0 " (whitespace in the end).